src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -1,647 +1,560 @@
 package com.minecolonies.entity.ai.citizen.farmer;
 
+import com.minecolonies.blocks.BlockHutField;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.Field;
 import com.minecolonies.colony.buildings.BuildingFarmer;
 import com.minecolonies.colony.jobs.JobFarmer;
+import com.minecolonies.entity.EntityCitizen;
 import com.minecolonies.entity.ai.basic.AbstractEntityAIInteract;
-import com.minecolonies.util.BlockPosUtil;
+import com.minecolonies.entity.ai.util.AIState;
+import com.minecolonies.entity.ai.util.AITarget;
+import com.minecolonies.util.BlockUtils;
 import com.minecolonies.util.InventoryUtils;
-import net.minecraft.block.Block;
+import net.minecraft.block.BlockCarrot;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.block.BlockCarrot'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)